samples/client/petstore/csharp/restsharp/net4.7/MultipleFrameworks/src/Org.OpenAPITools/Client/Configuration.cs

```diff
@@ -216,9 +216,9 @@ public virtual IDictionary<string, string> DefaultHeader
         public virtual IDictionary<string, string> DefaultHeaders { get; set; }
 
         /// <summary>
-        /// Gets or sets the HTTP timeout (milliseconds) of ApiClient. Default to 100000 milliseconds.
+        /// Gets or sets the HTTP timeout of ApiClient. Defaults to 100 seconds.
         /// </summary>
-        public virtual int Timeout { get; set; }
+        public virtual TimeSpan Timeout { get; set; }
```

Comment:
if users call the getter of Timeout in their code, they will need to update their code accordingly updating the C# client to a newer version including this change, right?

Reply:
Correct, if they use the 7.10.0 version of the generator to generate their client, they will need to update usages of Timeout.